src/Juvix/Compiler/Core/Translation/FromInternal.hs

```diff
@@ -719,14 +721,19 @@ goApplication a = do
 
   case f of
     Internal.ExpressionIden (Internal.IdenAxiom n) -> do
-      axiomInfo <- HashMap.lookupDefault impossible n <$> asks (^. Internal.infoAxioms)
-      case axiomInfo ^. Internal.axiomInfoBuiltin of
+      axiomInfoBuiltin <- getAxiomBuiltinInfo n
+      case axiomInfoBuiltin of
         Just Internal.BuiltinNatPrint -> app
         Just Internal.BuiltinStringPrint -> app
         Just Internal.BuiltinBoolPrint -> app
         Just Internal.BuiltinString -> app
         Just Internal.BuiltinIO -> app
-        Just Internal.BuiltinIOSequence -> app
+        Just Internal.BuiltinIOSequence -> do
+          as <- exprArgs
+          case as of
+            (arg1 : arg2 : xs) ->
+              return (mkApps' (mkConstr' (BuiltinTag TagBind) [arg1, mkLambda' (shift 1 arg2)]) xs)
+            _ -> error ">> must be called with 2 arguments"
```

Comment:
iirc `io-sequence` is not necessarily called `>>`. We should use the proper registered string.

The same goes for `||`, `&&`, etc.

Reply:
Let's address this when we do proper error checking earlier in the pipeline? I think these errors here should ultimately only be internal - the correct application arity should've been checked earlier.